doc/api/globals.md

```diff
@@ -235,4 +248,6 @@ The WHATWG `URLSearchParams` class. See the [`URLSearchParams`][] section.
 [buffer section]: buffer.html
 [built-in objects]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects
 [module system documentation]: modules.html
+[webassembly-mdn]: https://developer.mozilla.org/en-US/docs/WebAssembly
```

Comment:
Nit: it seems all these bottom references are sorted in ASCII order, so these two need to be placed after `[timers]: timers.html`

Reply:
Thanks, I fixed it 🔧